base_classes/NXdata.nxdl.xml

```diff
@@ -115,6 +115,38 @@
 			the axes attribute can be omitted.
 		</doc>
 	</attribute>
+	<attribute name="default_slice">
+		<doc>
+			Which slice of data to show in a plot by default. This is useful especially for
+			datasets with more than 2 dimensions.
+
+			Should be an array of strings of length equal to the number of dimesions
+			in the data, with the following possible values:
+			* ".": All the data in this dimension should be included
+			* Integer: Only this slice should be used.
+			* String: Only this slice should be used. Use if AXISNAME_indices is a string
+			array.
+
+			Example::
+
+				data: NXdata
+					@signal = "data"
+					@axes = ["image_id", "channel", ".", "."]
+					@image_id_indices = 0
+					@channel_indices = 1
+					@default_slice = [".", "difference", ".", "."]
+					image_id = [1, ..., nP]
+					channel = ["threshold_1", "threshold_2", "difference"]
+					data = uint[nP, nC, i, j]
```

Comment:
```suggestion
			  data:NXdata
				@signal = "data"
				@axes = ["image_id", "channel", ".", "."]
				@image_id_indices = 0
				@channel_indices = 1
				@default_slice = [".", "difference", ".", "."]
				image_id = [1, ..., nP]
				channel = ["threshold_1", "threshold_2", "difference"]
				data = uint[nP, nC, i, j]
```
I would suggest changing the spacing like this so it matches the other two examples in NXdata.